packages/mds-ingest-service/repository/migrations/1646177244396-DropMigrationIndices.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class DropMigrationIndices1646177244396 implements MigrationInterface {
```

Comment:
I just noticed, this needs to be included in `mds-ingest-service/repository/migrations/index.ts` too, or else it won't run!